lib/rollbax/item.ex

```diff
@@ -94,10 +95,10 @@ defmodule Rollbax.Item do
   end
 
   defp put_custom(data, custom) do
-    if map_size(custom) == 0 do
-      data
+    if map_size(custom) == 0 and map_size(data["custom"]) == 0 do
+      Map.delete(data, "custom")
```

Comment:
Instead of always adding `"custom"` and deleting it here in case `data["custom"]` is empty, let's not add `"custom"` in `draft/3` in case there's no custom data and do something like

```elixir
if map_size(custom) == 0 do
  data
else
  Map.update(data, "custom", custom, &Map.merge(&1, custom))
end
```

here. Wdyt?